scripts/run-operator.sh

```diff
@@ -11,30 +11,15 @@ cd .. || exit
 
 . "scripts/common.sh"
 
-GREEN='\033[0;32m'
-RED='\033[0;31m'
-NC='\033[0m' # No Color
-function info {
-  printf "${GREEN}run-operator.sh: ${1}${NC}\n"
-}
-function error {
-  printf "${RED}run-operator.sh: ${1}${NC}\n"
-}
-
 if ! minikube status
 then
   info "starting minikube"
   minikube start
 fi
 
-GIT_HASH="$(git rev-parse HEAD)"
 TMP_INSTALL_DIR="$(mktemp -d)"
 trap 'rm -rf -- "$TMP_INSTALL_DIR"' EXIT
 
-info "building operator image in minikube for commit ${GIT_HASH}"
-IMAGE_TAG="dev-git-${GIT_HASH}"
```

Comment:
now we have to pass IMAGE_TAG as an env var like `IMAGE_TAG=quay.io/robeyoun/operator:dev-git-c7976a9745445ee53440a4b49cfc97a8d280a76a ./scripts/run-operator.sh` could we make the script fail fast if this variable is blank?

Reply:
I'm tempted to say we do `set -euo pipefail` right at the start of the script. But I suppose that doesn't handle `IMAGE_TAG=''` Yeah I guess we can exit early

Reply:
after having a play there's also the awkwardness that we pass the `QUAY_ORG` to the build script, but then in `run-operator.sh` it's hardcoded to `quay.io/kroxylicious/operator`

Reply:
This should be resolved in the latest round of script wragling.